inspectors/models/notation/fretdiagrams/fretdiagramsettingsmodel.h

```diff
@@ -0,0 +1,59 @@
+#ifndef FRETDIAGRAMSETTINGSMODEL_H
+#define FRETDIAGRAMSETTINGSMODEL_H
+
+#include "models/abstractinspectormodel.h"
+
+#include "libmscore/fret.h"
+#include "libmscore/score.h"
+
+class FretDiagramSettingsModel : public AbstractInspectorModel {
+    Q_OBJECT
+
+    Q_PROPERTY(PropertyItem* strings READ strings CONSTANT)
+    Q_PROPERTY(PropertyItem* frets   READ frets   CONSTANT)
+    Q_PROPERTY(PropertyItem* showNut READ showNut CONSTANT)
+    Q_PROPERTY(PropertyItem* offset  READ offset  CONSTANT)
+    Q_PROPERTY(PropertyItem* numPos  READ numPos  CONSTANT)
+
+public:
+    explicit FretDiagramSettingsModel(QObject* parent, IElementRepositoryService* repository);
+
+    void createProperties() override;
+    void requestElements() override;
+    void loadProperties() override;
+    void resetProperties() override;
+
+    PropertyItem* strings() const;
+    PropertyItem* frets() const;
+    PropertyItem* showNut() const;
+    PropertyItem* offset() const;
+    PropertyItem* numPos() const;
+
+    Q_INVOKABLE bool showFretCanvas() const;
+
+    Ms::FretDiagram* fretDiagram() const;
+    Ms::FretDiagram* fretDiagram();
+    Q_INVOKABLE int dot(int s, int f) const;
+    Q_INVOKABLE int marker(int s) const;
+    Q_INVOKABLE bool barreExists(int f) const;
+    Q_INVOKABLE int barreStartString(int f) const;
+    Q_INVOKABLE int barreEndString(int f) const;
+    Q_INVOKABLE qreal barreLineWidth() const;
+
+public slots:
+    void setDot(int s, int f, bool add, int dType);
```

Comment:
s - seconds? s - strings? s - solars? :)

prefer to use meaningful names for params